buildstockbatch/postprocessing.py

```diff
@@ -107,12 +111,30 @@ def read_out_osw(fs, filename):
         ]
         for key in keys_to_copy:
             out_d[key] = d.get(key, None)
-        for step in d.get('steps', []):
-            if step['measure_dir_name'] == 'BuildExistingModel':
-                out_d['building_id'] = step['arguments']['building_id']
         return out_d
 
 
+def read_job_files(fs, started, finished):
+    jobs = {'completed_status': 'Fail'}
+    try:
+        with fs.open(started, 'r') as f:
+            started_at = re.search(r'Started Workflow (.*\s.*?)\s', f.readline()).group(1)
+            started_at = started_at.replace('-', '').replace(':', '').replace(' ', 'T') + 'Z'
+            jobs['started_at'] = started_at
+    except:
+        return None
+    try:
+        with fs.open(finished, 'r') as f:
+            completed_at = re.search(r'Finished Workflow (.*\s.*?)\s', f.readline()).group(1)
+            completed_at = completed_at.replace('-', '').replace(':', '').replace(' ', 'T') + 'Z'
```

Comment:
@joseph-robertson Is there really not a more python-ic way to convert from the one time string to the other?

Reply:
https://docs.python.org/3/library/time.html

Reply:
Or more specifically...

https://docs.python.org/3/library/datetime.html#strftime-and-strptime-behavior

Reply:
👀